tests/lib/rules/indent.js

```diff
@@ -1428,7 +1441,7 @@ ruleTester.run("indent", rule, {
         {
             code: fixture,
             output: fixedFixture,
-            options: [2, {SwitchCase: 1}],
+            options: [2, {SwitchCase: 1, MemberExpression: 1}],
```

Comment:
Why there is sometimes spaces after `{`/`}` and sometimes not? 
--
This remark shouldn't block this PR